app/public/scripts/script.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function() {
+
+  function refreshReceipt() {
+    $.get('/api/order/1', function(receiptData) {
+
+      T.render('receiptTemplate', function(receiptTemplate) {
+        $('#receipt-container').html( receiptTemplate(receiptData) );
```

Comment:
Nice using a template, makes this really tidy, although calling it `script.js` doesn't really explain to me what this is trying to do!